HerPublicWebsite.BusinessLogic/Services/CsvFileCreator/CsvFileCreator.cs

```diff
@@ -51,7 +51,7 @@ public MemoryStream CreateConsortiumReferralRequestFollowUpFileData(IEnumerable<
                     var consortiumStatistics = new ConsortiumStatistics(consortiumReferrals);
                     return new CsvRowConsortiumDownloadInformationRow(groupingByConsortium.Key, consortiumStatistics);
                 }
-            );
+            ).Where(row => !string.IsNullOrEmpty(row.Consortium)); //Do not include LAs which are not part of a consortium
```

Comment:
This should be on a new line, in the interests of readability. You should be able to get Rider to auto-format your code by highlighting it and pressing `Ctrl+Alt+Enter` (that is assuming your keybindings are the same as mine - can go to Help > Find Action... > Reformat Code to check). I'd only do that for code you've touched though, otherwise there will be loads of changes!

Reply:
Having said that, in this case we can save ourselves some work by moving this check earlier before the `Select`:
```
.Where(groupingByConsortium => groupingByConsortium.Key is not null)
```

Reply:
I didn't realise you could do the auto-reformat just for a selection! I'm going to be using that all the time now